docs/useGetRecordRepresentation.md

```diff
@@ -9,6 +9,13 @@ Get a function that returns the record representation, leveraging the [`<Record
 
 You can also use the component version: [`<RecordRepresentation>`](./RecordRepresentation.md).
 
+If you don't specify anything, the `useGetRecordRepresentation` function will choose the column to return according to this preference order:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** From my point of view this is not clear:: "If you don't specify anything", but where?
We could explain that this behavior occurs when `<Resource recordRepresentation>` is not defined. It makes more in my opinion.
Additionally, I suggest adding a mention in the `<RecordRepresentation>` doc, to explain that this component uses the `useRecordRepresentation` hook and that the same rules are therefore applied.
‏
‏

Reply:
https://marmelab.com/react-admin/useGetRecordRepresentation.html#usage